src/components/NavBar.tsx

```diff
@@ -20,6 +21,7 @@ const BottomTabs = () => {
         name="Home"
         component={HomeScreen}
         options={{
+          tabBarLabel: <TextRegular style={{ fontSize: 13 }}>Home</TextRegular>,
```

Comment:
I'm getting a type error `Type 'Element' is not assignable to type 'string'.t` here, but I can see text changes just fine. I haven't found a way to ignore the error (changing type definitions of the package didn't work), so if you can't find a workaround I can just merge this!

Reply:
I don't see the error on my end!